design-system/materials/internals/definition.yaml

```diff
@@ -809,6 +841,10 @@ decisionGroupsByTheme:
           choice: '0 var(--spacing-10)'
         padding-for-tooltip:
           choice: 'var(--spacing-10) var(--spacing-20)'
+        padding-for-view-switcher:
+          choice: '0 var(--spacing-30) 0 var(--spacing-30)'
+        padding-for-view-switcher-when-condensed:
+          choice: '0 var(--spacing-20) 0 var(--spacing-20)'
```

Comment:
Horizontal padding for each button when condensed should be 12px according to the [Miro board](https://miro.com/app/board/uXjVPSlrfYY=/?moveToWidget=3458764542513986030&cot=14) but currently is 8px.

Reply:
Nice catch. I actually added the token but forgot to update the component. 
Heres the commit -> [2688827](https://github.com/commercetools/ui-kit/pull/2390/commits/2688827749ddfa673a5793503cc7145f164531ed)
Thanks.

Reply:
But I mean, `spacing-20` is a token with the value `8px` and in the Miro board it says we should be using `12px` for horizontal spacing when condensed.

Reply:
This choice is for the default theme though. The default theme uses 8px for padding when condensed.

Reply:
My bad. Sorry 🤦